jenkins/ci.suse.de/cloud-crowbar-sap-testbuild-pr-trigger.yaml

```diff
@@ -0,0 +1,91 @@
+- job:
+    name: 'cloud-crowbar-sap-testbuild-pr-trigger'
+    node: cloud-trigger
+
+    parameters:
+      - choice:
+          name: mode
+          choices:
+            - normal
+            - rebuild
+            - forcerebuild
+            - forceall
+          description: |
+              normal: trigger unseen PRs
+              rebuild: trigger unseend and pending PRs
+              forcerebuild: trigger unseen, pending and failed PRs
+              forceall: trigger rebuild for all open PRs
+
+    triggers:
+      - timed: 'H/20 * * * *'
+
+    logrotate:
+      numToKeep: 48
+      daysToKeep: -1
+
+    builders:
+      - shell: |
+          ## THIS IS A TRIGGER JOB ONLY - NO WORKER CODE IN HERE - DO NOT DARE TO ADD SOME
+          ## TRIGGER JOBS ARE MOVED TO A DEDICATED TRIGGER NODE
+          ## THIS JOB IS ABOUT PRs FOR THE CCT REPO
+          [[ $NODE_NAME != 'cloud-trigger' ]] && exit 99
+          ##
+
+          set -x
+
+          export automationrepo=~/github.com/SUSE-Cloud/automation
+
+          # automation bootstrapping
+          if ! [ -e ${automationrepo}/scripts/jenkins/update_automation ] ; then
+              rm -rf ${automationrepo}
+              curl https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/jenkins/update_automation | bash
+          fi
+          # fetch the latest automation updates
+          ${automationrepo}/scripts/jenkins/update_automation
+
+          ghs=${automationrepo}/scripts/github-status/github-status.rb
+
+          action=list-unseen-prs
+          case $mode in
+            rebuild) action=list-rebuild-prs
+            ;;
+            forcerebuild) action=list-forcerebuild-prs
+            ;;
+            forceall) action=list-open-prs
+            ;;
+          esac
+
+
+          # PR testing for crowbar repos
+          declare -a crowbar_repos=(
+              crowbar
+              crowbar-ceph
+              crowbar-core
+              crowbar-ha
+              crowbar-openstack
+              crowbar-monitoring
+          )
+
+          declare -a master_branches=(
+              stable/3.0/sap
```

Comment:
I just realized that this is a illegal branch name. So we need to find something new.